packages/utilities/exposure-scanning/args/index.js

```diff
@@ -0,0 +1,45 @@
+const isValidId = id => {
+  try {
+    return !Number.isNaN(id) && Number.isInteger(parseFloat(id));
+  } catch {
+    return false;
+  }
+};
+
+const isValidRegex = regex => {
+  try {
+    RegExp(regex);
+  } catch {
+    return false;
+  }
+  return regex !== '';
+};
+
+const parseArgs = argv => {
+  if (argv.length !== 6) {
+    throw new Error(
+      'Incorrect number of args.\nUsage: node path/to/exposure-scanning <repo> <-pr/-issue> <id> <regex>',
+    );
+  }
+
+  const repo = argv[2];
+  const flag = argv[3]; // -pr or -issue
+  const id = argv[4];
+  const regexString = argv[5];
+
+  if (!isValidRegex(regexString)) {
+    throw new Error('Invalid regex argument given.');
+  }
+
+  if (!isValidId(id)) {
+    throw new Error('Invalid issue/pr id.');
```

Comment:
would this ever happen?

Reply:
Yeah if a non-integer argument is supplied then this should throw. It's unlikely to happen once integrated with the GitHub action, but since it's a psammead package and may be run by others outside of GitHub actions I think it may be necessary